msteams-platform/concepts/device-capabilities/location-capability.md

```diff
@@ -1,37 +1,54 @@
 
-The main advantage of integrating location capabilities in your Teams apps is that it allows web app developers on Teams platform to leverage location functionality with Microsoft Teams JavaScript client SDK.
+* Share authentic health data of cellular towers with the management. The management can compare any mismatch between captured location information and the data submitted by maintenance staff.
+* Locate technical support staff in a specified area.  The app asks support staff to share their current location, which management can use to allocate IT ticket to the nearest support person after checking their profile.
+* Report the location after completing a job in the field. The job processing app asks for permission to find the location. After the user grants permission, the app detects the exact location. The user may also select a location by dragging a pin to the job completion location over the map.
+* Capture attendance through selfies inside the retail store. The store manager can track the attendance of the workers by asking them to share a selfie through an attendance app. The location data gets captured and is sent along with the image. This scenario is applicable mainly to the frontline workers.
 
-Following examples show how the integration of location capabilities is used in different scenarios:
+# [Mobile](#tab/mobile)
 
-* In a factory, the supervisor can track the attendance of workers by asking them to take a selfie in the vicinity of the factory and share it through the specified app. The location data also gets captured and sent along with the image.
-* The location capabilities enable the maintenance staff of a service provider to share authentic health data of cellular towers with the management. The management can compare any mismatch between captured location information and the data submitted by maintenance staff.
+The following image depicts web app experience of getLocation API:
 
-To integrate location capabilities, you must update the app manifest file and call the APIs. For effective integration, you must have a good understanding of [code snippets](#code-snippets) for calling the location APIs.
-It's important to familiarize yourself with the [API response errors](#error-handling) to handle the errors in your Teams app.
+  <!-- ![Mobile app experience for location capabilities](../../assets/images/tabs/location-picker-mobile.png) -->
+  
+  :::image type="content" source="~/assets/images/tabs/location-picker-mobile.png" alt-text="Illustration shows the location picker." border="true":::
+
+# [Desktop](#tab/desktop)
+
+The following image depicts web app experience of getLocation API:
 
 > [!NOTE]
-> Currently, Microsoft Teams supports for location capabilities is available for mobile clients only.
+> The desktop support for `getLocation` and  `showLocation` is available after the user grants the device permission.
+
```

Comment:
Let's remove this note. Not required here as it's implicitly applicable to all APIs.